ambry-frontend/src/test/java/com.github.ambry.frontend/FrontendIntegrationTest.java

```diff
@@ -561,7 +565,7 @@ private static VerifiableProperties buildFrontendVProps(File trustStoreFile)
     TestSSLUtils.addSSLProperties(properties, "", SSLFactory.Mode.SERVER, trustStoreFile, "frontend");
```

Comment:
Could you also change uploadDataChunksAndVerify to add the prefixes to signedChunkIds